frontend/src/components/Form.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react'
+import { FormField } from './FormField'
+
+export interface FormData {
+  name: string
+  age: string
+  favoriteColor: string
+  preferredContactMethod: string
+}
+
+export const Form = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    age: '',
+    favoriteColor: '',
+    preferredContactMethod: 'Email',
+  })
+  const contactMethods = ['Email', 'Phone']
```

Comment:
It's funny to see contact methods but no input for email/phone. I guess if it wasn't mentioned explicitly in the task, it's not needed (=